docs/join-the-network/run-avail/other-nodes/0050-avail-bootstrap-node.md

```diff
@@ -0,0 +1,60 @@
+---
+id: avail-bootstrap-node
+title: Run a Bootstrap Node
+sidebar_label: Run a Bootstrap Node
+description: Learn how to run a boostrap node
+keywords:
+  - docs
+  - avail
+  - node
+  - data availability
+  - da
+image: https://availproject.github.io/img/avail/AvailDocs.png
+slug: avail-bootstrap-node
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Bootstrap Node Setup
+
+If for some reason you ever wanted to host your own network, you will need a node that will serve as an access point to all the other nodes wanting to join.
+This is where the Bootstrap node comes onto to the stage. We offer a simple and functioning setup for a such node, just be sure to deploy few of these and that they are publicly accessible over the network.
+
+First, binary for the Bootstrap node needs to be compiled. On your local machine, or either on any cloud provider VM, be sure that Rust is installed:
+```
+curl https://sh.rustup.rs -sSf | sh
+source $HOME/.cargo/env
+rustup update nightly
+rustup target add wasm32-unknown-unknown --toolchain nightly
+rustc --version # verify rust is working, print the installed version
+```
```

Comment:
Might need to install some compiler packages if its a new ubuntu machine. I ran this `apt install build-essential`